admin/assets/javascripts/jquery-modal.js

```diff
@@ -7,12 +7,18 @@
 $(function () {
   function startModal (e) {
     $('.modal-link').on('click', function (e) {
-      $('#js-modal-confirmation-button').attr('href', $(this)[0].href)
+      if ($('#js-modal-confirmation-button').is('button')) {
```

Comment:
so this now works with both layouts i take it?

Reply:
@GuyHarwood that is correct.
Will amend accordingly when all the relevant usages are moved to the new toolkit